tnt/tnt_io.c

```diff
@@ -138,15 +139,13 @@ tnt_io_connect_do(struct tnt_stream_net *s, struct sockaddr *addr,
 						return TNT_ESYSTEM;
 					}
 					/* calculate timeout last time */
-					int64_t passd_usec = (curr.tv_sec - start_connect.tv_sec) * micro +
-						(curr.tv_usec - start_connect.tv_usec);
-					int64_t curr_tmeout = passd_usec - tmout_usec;
-					if (curr_tmeout <= 0) {
+					int passd_msec = (curr.tv_sec - start_connect.tv_sec) * milli +
```

Comment:
Nit: >80 symbols.

Reply:
The line, that i replaced was > 80 but < 120 simbols too. Please suggest other way.

Reply:
Our C code style suggests to limit line length to 80 symbols: [1]. If the previous code was not formatted properly, so it is bad, but is not the reason to break code style in the new code.

[1]: https://tarantool.io/en/doc/1.9/dev_guide/c_style_guide/#chapter-2-breaking-long-lines-and-strings

Reply:
i mean, i do not know how to make line < 80 symbols gracefully

Reply:
It likely will not be beautiful. Ideally we should decrease indent level by extracting code pieces into functions. But this is a kind of refactoring and usually is not mandatory (or even discouraged if is massive) within a feature/bug scope.

Don't worry much about that, this is the small piece of the code.

Reply:
Thank you
Fixed